### ISSUE TITLE
create a script to bring up the infra compoenents of global scheduler platform (API server instances, Distributor instances, Scheduler instances, Dispatcher instances etc.) 

### ISSUE THREAD

**Issue by cathyhongzhang** [futurewei-cloud/global-resource-scheduler#85]
**What would you like to be added**:
We decide to use scripts to create instances of these platform compoenents, similar to arktos-up.sh
Also needs to add HA hook for each active instance so that when the active leader instance is crashed, one follower instance can assume the leader role.

**Comment by jshaofuturewei** [comment]
20% completed

I set up a script to run kube-apiserver, kube-scheduler and kuberlet which are necessary to run global scheduler now.

The kubelet and kubescheduler shall be removed once the light weighted scheduler from HQ is merged.

**Comment by jshaofuturewei** [comment]
25% completed.

Added kim's command to start global resource scheduler.

**Comment by cathyhongzhang** [comment]
work to be done:
1. add distributor to the startup script
2. add lightweight scheduler to the script
3. add dispatcher to startup script
4. talk with other members in open source to see if the resource collector is a process that needs to be started by the script.

**Comment by CoderKevinZhang** [comment]
Need to merge into master first, and then run integration to see if components can be matched up
